packages/bpk-mixins/src/mixins/_chips.scss

```diff
@@ -24,143 +24,65 @@ $bpk-spacing-v2: true;
   padding: 0 bpk-spacing-base();
   align-items: center;
   border: none;
+  border-radius: $bpk-border-radius-xs * 2;
 
-  @include bpk-border-radius-pill;
+  &--has-trailing {
+    padding-right: bpk-spacing-md();
+  }
 
-  &--light {
-    background-color: $bpk-color-white;
-    color: $bpk-color-sky-gray;
+  &--on-dark {
+    background-color: transparent;
+    color: $bpk-text-on-dark-day;
 
-    @include bpk-border-sm($bpk-color-sky-gray-tint-03);
+    @include bpk-border-sm($bpk-line-on-dark-day);
 
     @include bpk-hover {
-      background-color: $bpk-color-sky-gray-tint-07;
-    }
-
-    &:active:not(:disabled) {
-      background-color: $bpk-color-sky-gray-tint-06;
+      @include bpk-border-sm($bpk-surface-elevated-day);
     }
 
     &-selected {
       @include bpk-themeable-property(
         color,
-        --bpk-chip-light-selected-text-color,
-        $bpk-color-white
+        --bpk-chip-on-dark-selected-text-color,
+        $bpk-text-primary-day
```

Comment:
In Figma this is actually `$bpk-text-on-light-day` 🙂

Reply:
[The figma ](https://www.figma.com/file/kX4UHoNVZQUDeDMjPeyLOw/Chips-and-Badges?node-id=618%3A16174)documentation I'm reading is probably out of date, resulting in the wrong token being used. I will modify the wrong token according to the latest figma document.